compiler/luci-interpreter/src/kernels/LogSoftmax.cpp

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "kernels/LogSoftmax.h"
+
+#include "kernels/Utils.h"
+
+#include <tensorflow/lite/kernels/internal/reference/reference_ops.h>
+
+#include <tensorflow/lite/kernels/internal/optimized/optimized_ops.h>
+
+namespace luci_interpreter
+{
+namespace kernels
+{
+
+LogSoftmax::LogSoftmax(const Tensor *input, Tensor *output) : Kernel({input}, {output}) {}
+
+void LogSoftmax::configure()
+{
+  assert(input()->element_type() == output()->element_type());
```

Comment:
Please use `LUCI_INTERPRETER_CHECK` instead of assert.

`LUCI_INTERPRETER_CHECK` will throw an exception if condition is not true.

Reply:
Please add more checks.

LogSoftmax has restriction on quantization parameters. (scale, zp) should be (16.0 / 256.0, 127).
https://www.tensorflow.org/lite/performance/quantization_spec

Reply:
Thank you for you feedback, @jinevening !
I will change assert to LUCI_INTERPRETER_CHECK!

Reply:
Added checks for scale & zero_point.
Thank you for reviewing!